lib/FFI/Platypus/Type/Enum.pm

```diff
@@ -278,6 +292,9 @@ sub ffi_custom_type_api_1
   $config{rev} ||= 'str';
   ($config{rev} =~ /^(int|str|dualvar)$/) or croak("rev must be either 'int', 'str', or 'dualvar'");
 
+  $config{casing} ||= 'upper';
+  ($config{casing} =~ /^(upper|keep)$/) or croak("casing must be either 'upper' or 'keep'");
```

Comment:
:+1: this allows us to add `lower` or others later and user gets the diagnostic.